### PR TITLE
Completed prep-javascript-dom-manipulation.

### PR DESCRIPTION
**Pull request by bookaneers**
Fernando Figueiredo | prep-javascript-dom-manipulation
Screen Shot 2021-02-26 at 7 00 46 PM

### CODE REVIEW

File: prep-javascript-dom-manipulation/index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <p class="third-paragraph">
+      I hope one day to meet people that also like coding, television, and pancakes.
+    </p>
+  </div>
+  <script>
+    // Your code here
+    $secondParagraph = document.querySelector('p.second-paragraph');
+    console.log($secondParagraph);
+
+    $thirdParagraph = document.querySelector('p.third-paragraph');
```
Comment (thebearingedge) [inline prep-javascript-dom-manipulation/index.html]: Be sure to use `var` when you declare your variables.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Great, just one small adjustment needed.

**Review by thebearingedge — APPROVED** [review]
👍

**Review by bookaneers — COMMENTED** [review]
Fernando Figueiredo | prep-javascript-dom-manipulation
Screen Shot 2021-02-27 at 12 06 32 PM

**Pull request merged**